test/sync_to_md.md.expected

```diff
@@ -37,6 +37,7 @@ let () =
   ()
 ;;
 
+
```

Comment:
why is there extra new lines in the expected files?

Reply:
Fixed.
Part parsing no longer remove empty lines at the end of parts, this was a bug in the code that update md files.